TestCases/engine/marker_engine_inflow/inv_channel_engine_inflow.cfg

```diff
@@ -0,0 +1,221 @@
+%
+% Marker(s) of the surface to be plotted or designed
+MARKER_PLOTTING= ( lower_wall )
+%
+% Marker(s) of the surface where the functional (Cd, Cl, etc.) will be evaluated
+MARKER_MONITORING= ( upper_wall, lower_wall )
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= GREEN_GAUSS
+%
+% Courant-Friedrichs-Lewy condition of the finest grid
+CFL_NUMBER= 50.0
+%
+% Adaptive CFL number (NO, YES)
+CFL_ADAPT= YES
+%
+% Parameters of the adaptive CFL number (factor down, factor up, CFL min value,
+%                                        CFL max value )
+CFL_ADAPT_PARAM= ( 0.1, 2.0, 50.0, 1e10 )
+%
+% Runge-Kutta alpha coefficients
+RK_ALPHA_COEFF= ( 0.66667, 0.66667, 1.000000 )
+%
+% Number of total iterations
+ITER= 999999
+
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+%
+% Linear solver for implicit formulations (BCGSTAB, FGMRES)
+LINEAR_SOLVER= FGMRES
+%
+% Preconditioner of the Krylov linear solver (ILU, JACOBI, LINELET, LU_SGS)
+LINEAR_SOLVER_PREC= ILU
+%
+% Minimum error of the linear solver for implicit formulations
+LINEAR_SOLVER_ERROR= 1E-10
+%
+% Max number of iterations of the linear solver for the implicit formulation
+LINEAR_SOLVER_ITER= 20
+
+% -------------------------- MULTIGRID PARAMETERS -----------------------------%
+%
+% Multi-Grid Levels (0 = no multi-grid)
+MGLEVEL= 3
+%
+% Multi-grid cycle (V_CYCLE, W_CYCLE, FULLMG_CYCLE)
+MGCYCLE= W_CYCLE
+%
+% Multi-grid pre-smoothing level
+MG_PRE_SMOOTH= ( 1, 2, 3, 3 )
+%
+% Multi-grid post-smoothing level
+MG_POST_SMOOTH= ( 0, 0, 0, 0 )
+%
+% Jacobi implicit smoothing of the correction
+MG_CORRECTION_SMOOTH= ( 0, 0, 0, 0 )
+%
+% Damping factor for the residual restriction
+MG_DAMP_RESTRICTION= 1.0
+%
+% Damping factor for the correction prolongation
+MG_DAMP_PROLONGATION= 1.0
+
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+%
+% Convective numerical method (JST, LAX-FRIEDRICH, CUSP, ROE, AUSM, HLLC,
+%                              TURKEL_PREC, MSW)
+CONV_NUM_METHOD_FLOW= JST
+%
+% 2nd and 4th order artificial dissipation coefficients
+JST_SENSOR_COEFF= ( 0.5, 0.02 )
+%
+% Time discretization (RUNGE-KUTTA_EXPLICIT, EULER_IMPLICIT, EULER_EXPLICIT)
+TIME_DISCRE_FLOW= EULER_IMPLICIT
+
+% --------------------------- CONVERGENCE PARAMETERS --------------------------%
+%
+% Convergence criteria (CAUCHY, RESIDUAL)
+CONV_FIELD= RMS_DENSITY
+%
+% Min value of the residual (log10 of the residual)
+CONV_RESIDUAL_MINVAL= -10
+%
+% Start convergence criteria at iteration number
+CONV_STARTITER= 10
+%
+% Number of elements to apply the criteria
+CONV_CAUCHY_ELEMS= 100
+%
+% Epsilon to control the series convergence
+CONV_CAUCHY_EPS= 1E-10
+
+% ------------------------- INPUT/OUTPUT INFORMATION --------------------------%
+%
+% Mesh input file
+MESH_FILENAME= mesh_channel_256x128.su2
```

Comment:
Can you use one of the other channel meshes we have in the repo? you can use relative paths here.

Reply:
I changed it. Hopefully it will work this time.